src/cosmosdb-continuous-backup/README.md

```diff
@@ -0,0 +1,102 @@
+# Microsoft Azure CLI 'cosmosdb-continuous-backup' Extension #
+
+This package is for the 'cosmosdb-continuous-backup' extension.
+This package provides commands to
+
+- Create Azure CosmosDB continuous backup accounts
+- List the different versions of databases and collections that were modified
+- Trigger a point in time restore on the Azure CosmosDB continuous mode backup accounts.
+- Update the backup interval and backup retention of periodic mode bbackup accounts.  
+
+## How to use ##
+
+Install this extension using the below CLI command
+
+```sh
+az extension add --name cosmodb-continuous-backup
+```
+
+### Included Features ###
+
+#### Create a new CosmosDB continuous backup Account ####
+
+```sh
+az cosmosdb create \
+    --resource-group "my-rg" \
+    --name "my-continuous-backup-account" \
+    --backup-policy-type "Continuous"
+```
+
+#### List all the CosmosDB accounts that can be restored (live and deleted) ####
+
+This command returns all the continuous mode backup accounts that can be restored.
+
+```sh
+az cosmosdb restorable-database-account list
+```
+
+#### Restore from an existing(live or deleted) database account to a new account ####
+
+```sh
+az cosmosdb restore --resource-group "my-rg" \
+    --target-database-account-name "restored-account" \
+    --source-database-account-name "mysourceaccount" \
+    --restore-timestamp "2020-07-20T16:09:53+0000" \
+    --location "westus" \
+    --databases-to-restore name="MyDB1" collections="collection1" "collection2" \
+    --databases-to-restore name="MyDB2" collections="collection3" "collection4"
+```
+
+#### List all the versions of sql databases in a live database account ####
```

Comment:
nit: can we have different subsection for SQL and MongoDB?

Reply:
We can. But I felt it might just increase the length of the command, without adding much use. 
Lets discuss offline if we have to go with separate subsections